### PR TITLE
Add function comment to fix golint error in cmd/kube-scheduler/app.

### PR DESCRIPTION
**Pull request by tizhou86**
**What this PR does / why we need it**:
Add function comment to fix golint error in cmd/kube-scheduler/app.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by misterikkit — APPROVED** [review]
/lgtm
/ok-to-test
Thanks for the cleanup

**Comment by misterikkit** [comment]
haha, looks like you get to take this package off a blacklist.

> W0823 17:39:10.143] Some packages in hack/.golint_failures are passing golint. Please remove them.
> W0823 17:39:10.143] 
> W0823 17:39:10.143]   cmd/kube-scheduler/app

**Comment by tizhou86** [comment]
@misterikkit Fixed, thanks for pointing it out!

File: cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -122,6 +122,7 @@ through the API as necessary.`,
 	return cmd
 }
 
+// Run runs the Scheduler. This should never exit.
```
Comment (sttts) [inline cmd/kube-scheduler/app/server.go]: it does on stopCh close, no?

Comment (tizhou86) [reply cmd/kube-scheduler/app/server.go]: Revised, thanks for pointing it out!

**Comment by tizhou86** [comment]
Hi @sttts I've revised the comment, need your approval on this pr, thanks!

**Pull request merged**